### DISCUSSION TITLE
High resolution version of the miryoku-kle-cover.png

### DISCUSSION THREAD

**Discussion by mgorfer** [manna-harbour/miryoku#248] [Q&A] [answered]
I am just starting out in the world of custom keyboards and alternate layouts.

I would like to print out the miryoku-kle-cover.png as a poster, to help me remember the key locations.
The resolution is not that high, unfortunately, and it gets very pixelated.
Would it be possible to create a higher resolution cover picture?

Back then, when I was learning Japanese, I was delighted with my Kanji Wall Poster, 
and the Miryoku 8 layer journey really reminds me on that :) 
https://omgjapan.com/products/kanji-wall-poster

**Comment by manna-harbour** [comment, marked as answer]
See https://github.com/manna-harbour/miryoku/issues/72.

For making a high res version of the current diagram see https://github.com/manna-harbour/miryoku/issues/72#issuecomment-1029597857. 

A wall poster is a nice idea, thanks! For a per-layer SVG suitable for use in making a wall poster see https://github.com/manna-harbour/miryoku/issues/72#issuecomment-1154588220.

**Reply by mgorfer** [reply]
Thank you for the advice!

I created a screenshot of the Layer Reference shown in http://www.keyboard-layout-editor.com/#/gists/6a7b0c22545ec03a776d5f7c3d5d36a2

The (experimental) download to SVG does not work well, unfortunately, but the screenshot is quality good enough for my DIN A1 Poster plans.

Hope my speed of around 2 WPM will increase soon! :)

**Reply by manna-harbour** [reply]
Yes, SVG download from KLE doesn't work. See above for 2 other ways of obtaining SVGs.
